static/js/controllers/contacts.js

```diff
@@ -240,18 +236,25 @@ var _ = require('underscore'),
           $scope.filtered = true;
           liveList = LiveList['contact-search'];
           liveList.set([]);
-          _query();
+          return _query();
         } else {
           $scope.filtered = false;
-          _query();
+          return _query();
         }
       };
 
+      $scope.sort = function($event) {
+        $scope.sortDirection = $event.target.getAttribute('data-value');
```

Comment:
I've put a comment in the template below but I think it'd be cleaner to just hardcode the direction than to pass the $event and get it out of the DOM.